.github/workflows/docker.yml

```diff
@@ -11,7 +13,16 @@ jobs:
       - uses: actions/checkout@v2
       - name: Build the Docker image
         run: docker build .
-      - name: build-and-push
+      - name: Push Develop Image if on develop branch
+        if: ${{ github.event_name == 'push' && github.ref == 'develop' }}
```

Comment:
Looks like the only branch we're using is `develop`, do we still need to check `github.ref == 'develop'`?

Might not be necessary, but maybe it's a good safeguard in case anything changes in the future.

Reply:
Yeah in case someone changes then this GHA runs don't want to push a develop tag on a different branch :D